src/index.html

```diff
@@ -20,7 +20,8 @@
       <h1>Leaderboard</h1>
     </header>
     <main class="flex-container">
-      <div class="scores-section column">
+      <div class="row">
+      <div class="scores-section col-8 col-sm-6">
```

Comment:
- [ ] There is a commented code in `src/index.js line 1`, please check and remove it as comments make an otherwise clean code look unclean. You can use GitHistory to refer to previous versions of your code.

Reply:
Hi mike i have been stuck at this for hours "apiData.js:6 Uncaught ReferenceError: regeneratorRuntime is not defined at apiData.js:6"

the local enviroment seems to work fine after i fixed the babel, how ever deploying github pages it won't , it seems github won't install the dependencies.